app/uk/gov/hmrc/incometaxpenaltiesappealsfrontend/controllers/SingleAppealConfirmationController.scala

```diff
@@ -0,0 +1,70 @@
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.incometaxpenaltiesappealsfrontend.controllers
+
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import uk.gov.hmrc.incometaxpenaltiesappealsfrontend.config.ErrorHandler
+import uk.gov.hmrc.incometaxpenaltiesappealsfrontend.controllers
+import uk.gov.hmrc.incometaxpenaltiesappealsfrontend.controllers.predicates.{AuthAction, UserAnswersAction}
+import uk.gov.hmrc.incometaxpenaltiesappealsfrontend.services.UserAnswersService
+import uk.gov.hmrc.incometaxpenaltiesappealsfrontend.views.html._
+import uk.gov.hmrc.incometaxpenaltiesfrontend.controllers.predicates.NavBarRetrievalAction
+import uk.gov.hmrc.incometaxpenaltiesappealsfrontend.pages.SingleAppealConfirmationPage
+
+import javax.inject.Inject
+import scala.concurrent.{ExecutionContext, Future}
+
+
+class SingleAppealConfirmationController @Inject()(singleAppealConfirmationView: SingleAppealConfirmationView,
+                                      val authorised: AuthAction,
+                                      withNavBar: NavBarRetrievalAction,
+                                      withAnswers: UserAnswersAction,
+                                      userAnswersService: UserAnswersService,
+                                      override val errorHandler: ErrorHandler,
+                                      override val controllerComponents: MessagesControllerComponents
+                                     )(implicit ec: ExecutionContext) extends BaseUserAnswersController {
+
+
+//  def onPageLoad(): Action[AnyContent] = (authorised andThen withNavBar andThen withAnswers).async { implicit user =>
+//      Future(Ok(singleAppealConfirmationView("first", 300.00)))
+//  }
+
+  def onPageLoad(): Action[AnyContent] = (authorised andThen withNavBar andThen withAnswers) { implicit user =>
+
+    user.penaltyData.multiplePenaltiesData match {
+      case Some(multiplePenaltiesData) =>
+        Ok(singleAppealConfirmationView(
+          //TODO" add first/second appeal dynamically
+          "first",
+          multiplePenaltiesData.firstPenaltyAmount
+        ))
+      case _ =>
+        Redirect(controllers.routes.ReasonableExcuseController.onPageLoad())
+    }
+  }
+
+
+  def submit(): Action[AnyContent] = (authorised andThen withNavBar andThen withAnswers).async { implicit user =>
+
+    val updatedAnswers = user.userAnswers.setAnswer(SingleAppealConfirmationPage, true)
+    userAnswersService.updateAnswers(updatedAnswers).map { _ =>
+      //TODO: redirect
+      Redirect(routes.WhenDidEventHappenController.onPageLoad())
+    }
+  }
```

Comment:
We don't need to save a value for this page, as it's just one that we present the User for informational purposes only. 

So, the submit can just do the redirect. See Nadin's new page for an example of this 🙂